paddle/framework/.clang-format

```diff
@@ -0,0 +1,5 @@
+---
+Language:        Cpp
```

Comment:
why we need this new clang-format file? We only have one in `Paddle/.clang-format`.